src/lazy/binary/raw/v1_1/struct.rs

```diff
@@ -140,7 +140,7 @@ impl<'top> RawBinaryStructIterator_1_1<'top> {
             bytes_to_skip: 0,
             struct_type: match opcode_type {
                 // TODO: Delimited struct handling
-                OpcodeType::Struct => StructType::FlexSym,
+                OpcodeType::Struct => StructType::SymbolAddress,
```

Comment:
🗺️ For now, all structs are treated as `SymbolAddress` structs. We still need to add the switch-to-`FlexSym` functionality for length-prefixed structs.